packages/server/src/api/controllers/row/ExternalRequest.ts

```diff
@@ -572,11 +586,18 @@ export class ExternalRequest {
       if (!linkTable || !linkPrimary) {
         return
       }
+
+      const linkSecondary =
```

Comment:
This is solving an issue that occurred when reassigning many-to-many relationships